data/config-schema/src/main/resources/net/corda/schema/configuration/p2p.gateway/1.0/corda.p2p.gateway.json

```diff
@@ -41,6 +41,12 @@
         "additionalProperties": false
       }
     },
+    "maxRequestSize": {
+      "description": "The maximum request size accepted by the gateway, in bytes. Any requests crossing that threshold will be rejected.",
```

Comment:
Maybe we should use kb or mb (or at least set the default and minimum using a scientific notation (`500000000` -> `5e8`))?

Reply:
I think kb/mb etc. is not natively supported and I would be relucant to start adding custom parsing code. But, scientific notation seems to be supported. So, I can give that a try.

Reply:
I meant to change the description to: `... request size accepted by the gateway, in megabytes ...` (But I'm OK either way)